CONTRIBUTING.md

```diff
@@ -44,9 +44,9 @@ git clone git@github.com:determined-ai/determined.git
 - Go (>= 1.13)
 - Python (>= 3.6)
 - Node (>= 12)
-- Yarn (>= 1.22.0)
 - Docker (>= 19.03)
 - Protoc (>= 3.0)
+- Java (>= 7)
```

Comment:
question: Do we need to announce in engineering once this lands that they need to update? Is this just if you want to work on the front-end?

Reply:
hmm needed by anyone that needs to build the npm package, publish it, or otherwise use it so for the time being mostly anyone working touching the frontend or wanting to run frontend tests.

Reply:
Please announce in #engineering once this lands. Thanks!